contracts/Registry.sol

```diff
@@ -1,70 +1,604 @@
-pragma solidity ^0.4.21;
+pragma solidity ^0.4.24;
 
-import "./Agent.sol";
+import "./IRegistry.sol";
 
-contract Registry {
-    enum RecordState {
-        ALIVE,
-        DEAD
+contract Registry is IRegistry {
+
+    struct OrganizationRegistration {
+        bytes32 organizationName;
+        address owner;
+        mapping(address => bool) members;
```

Comment:
We're going to need to keep a *list* of members too so that we can delete them all from this map upon org deletion; else a new organization with the same name will inherit the member list of the prior (and have no way to determine who the members are)

On that note, we also need getters for organization members in my opinion.

Reply:
so Lon and I talked about this and sort of punted on the getters/view functions but since we definitely need this list of members to clean up then this becomes a bigger issue. will implement.